src/material.js

```diff
@@ -215,31 +214,31 @@ export class Material {
      * @param salt {String} Salt for hashing, empty string by default.
      * @param isScaled {Boolean} Whether to scale the lattice parameter 'a' to 1.
      */
-    calculateHash(salt = '', isScaled = false) {
```

Comment:
reverted function back to original. Moved chemicalHash creation to function on line 235